test/controllers/util/SubmissionDisplayHelperSpec.scala

```diff
@@ -217,11 +218,12 @@ object SubmissionDisplayHelperSpec {
   private lazy val functionCodes: Seq[String] =
     Seq("01", "02", "03", "05", "06", "07", "08", "09", "10", "11", "16", "17", "18")
   private lazy val functionCodesRandomised: Iterator[String] = Random.shuffle(functionCodes).toIterator
+
   private def randomResponseFunctionCode: String = functionCodesRandomised.next()
 
-  val dateTimeIssued: LocalDateTime = LocalDateTime.now()
-  val dateTimeIssued_2: LocalDateTime = dateTimeIssued.plusMinutes(3)
-  val dateTimeIssued_3: LocalDateTime = dateTimeIssued_2.plusMinutes(3)
+  val dateTimeIssued: Instant = Instant.now()
+  val dateTimeIssued_2: Instant = dateTimeIssued.plus(3, MINUTES);
+  val dateTimeIssued_3: Instant = dateTimeIssued_2.plus(3, MINUTES);
```

Comment:
Please remove semicolons :) It's Scala world! :)

Reply:
ohhh god!!!! No soup for me!!